### PR TITLE
Center aligned header

### LINKED ISSUE #2165: Aligning the Header in README.md
+ I suggest center aligning the image and the badges as follows
+ Already completed the restructuring and ready to send a PR

IMG :
image

### PR DESCRIPTION
**Pull request by the-amazing-atharva**
Fixes #2165 

#### Special notes for your reviewer:

-

### CODE REVIEW

**Review by kannon92 — DISMISSED** [review]

**Comment by kannon92** [comment]
You have conflicts in the README.  Could you fix that? After that it is good to go.

**Comment by the-amazing-atharva** [comment]
> You have conflicts in the README. Could you fix that? After that it is good to go.

Just resolved them
Is it proper now?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
+<div align="center">
 <img src="./logo.svg" width="200"/>
 
-[![CircleCI](https://circleci.com/gh/armadaproject/armada.svg?style=shield)](https://circleci.com/gh/armadaproject/armada)
```
Comment (kannon92) [inline README.md:3]: Please add back these badges

**Review by kannon92 — CHANGES_REQUESTED** [review]
Align center looks good but let’s please keep those badges

**Comment by the-amazing-atharva** [comment]
Sent a new PR, as I made an error while resolving (have not yet resolved a conflict b4)

**Pull request closed without merging**